int32  xAcceleration
int32  yAcceleration
int32  zAcceleration
int32  xMagnetic
int32  yMagnetic
int32  zMagnetic
int32  pitch
int32  roll
int32  heading
string comment
